### PR TITLE
Add nightly

### PR DESCRIPTION
**Pull request by markhermeling**
# Description

Please describe the scope of the fix or feature addition.

Fixes zd#

# Testing

How did you test?

# Checklist

 - [ ] added tests
 - [ ] updated/added doxygen
 - [ ] updated appropriate READMEs
 - [ ] Updated manual and documentation

### CODE REVIEW

**Comment by anhu** [comment]
Hi Mark, 
Once you think this branch is ready for merging into master, can you please send a message to support@wolfssl.com to request a contributor agreement? 

Warm regards, Anthony

File: .github/workflows/daily.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      ROOT_TREE: "WolfSSL-Projects/WolfSSL-GitHub"
+      PROJECT_NAME: "wolfssl"
+      BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
+      COMMIT_HASH: $GITHUB_SHA
+      CSONAR_HUB_URL: "https://partnerdemo.codesonar.com"
+      CSONAR_HUB_USER: "${{ secrets.CSONAR_HUB_USER }}"
+      CSONAR_HUB_PASSWORD: "${{ secrets.CSONAR_HUB_PASS }}"
+      CSONAR_CSHOME: /opt/codesonar
+      CSO_GITHUB: /opt/codesonar-github
+      REPO_URL: "http://github.com/markhermeling/wolfssl"    # the github.repositoryUrl is not an http-link.
```
Comment (JacobBarthelmeh) [inline .github/workflows/daily.yaml:35]: With this setup would this test have a dependency on the forked wolfssl repository?

**Review by JacobBarthelmeh — COMMENTED** [review]
Could you clean up the githistory on this? Squash the commits and rebase onto master.

File: .github/workflows/daily.yaml (hunk 1 above)
Comment (markhermeling) [reply .github/workflows/daily.yaml:35]: It does not. It merges from daily-snapshot into my own fork and runs the analysis there. I figured the separation would be nice.

**Pull request closed without merging**